src/content/docs/infrastructure/prometheus-integrations/integrations-list/node-exporter-integration.mdx

```diff
@@ -0,0 +1,418 @@
+    # Note that <VERSION>, <OS>, and <ARCH> are placeholders.
+    wget https://github.com/prometheus/node_exporter/releases/download/v<VERSION>/node_exporter-<VERSION>.<OS>-<ARCH>.tar.gz
+    tar xvfz node_exporter-*.*-amd64.tar.gz
+    cd node_exporter-*.*-amd64
+    ./node_exporter
+    ```
+2. Set Node Exporter to run in the background with the keyboard commands `CONTROL + z` and `bg`. In production environments, you'd want to set this up as a service (for example, with `systemd`).
+
+## Configurations [#configs]
+
+Before you start Prometheus, you'll need to make some changes in your main `prometheus.yml` configuration file. We'll start with this basic `prometheus.yml` example below and add configurations to it in the remaining sections. You can copy these examples and paste them into your configuration file.
+
+Note that `job_name` is set to `node`: 
+
+```yml lineHighlight=11
+# my global config 
+global:
+  scrape_interval: 15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  - job_name: node
+```
+
+### Connect Prometheus to New Relic [#add-url-and-license]
+
+In your `prometheus.yml`, insert the `remote_write` snippet from the example below. Keep the following in mind:
+
+* This is a snippet for Prometheus v2.26 and higher. If you're using an older version, see our main [remote write instructions](/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/#setup). 
+* Make sure to replace `YOUR_LICENSE_KEY` with your value. 
+* You can insert this at the bottom of the configuration file at same indentation level as the `global` section.
+
+```yml lineHighlight=12-15
+# my global config 
+global:
+  scrape_interval: 15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  - job_name: node
+
+remote_write:
+  - url: https://metric-api.newrelic.com/prometheus/v1/write?prometheus_server=NodeExporter
+    authorization:
+      credentials: YOUR_LICENSE_KEY
+```
+
+### Set up targets [#set-up-targets]
+
+You can configure targets statically via the `static_configs` parameter, or you can use dynamic discovery with one of the supported service-discovery mechanisms.  
+
+#### Static targets [#static-targets]
```

Comment:
Should these be steps? Can they be steps? Trying to make sense of the H4 vs H3-- are these all the ways you can/must configure?

Reply:
@homelessbirds, I was trying to set up a choice: you either configure the targets statically or you configure them dynamically. How could I make that clearer?

Reply:
What if you just said "Configure static targets" and "Configure dynamic targets"? It might be a little wordy but it feels clearer and more directive to me.